python/py-sqlparse/Portfile

```diff
@@ -4,7 +4,7 @@ PortSystem          1.0
 PortGroup           python 1.0
 
 name                py-sqlparse
-version             0.2.4
+version             0.3.0
```

Comment:
In the commit message, did you mean: `python/py-sqlparse`?

Reply:
Uh, yes, yes I did.

Reply:
Did you want to fix that, or do you want to merge as-is?

Reply:
Yes, I'll fix it later tonight!

Reply:
ready to merge.